src/contract/event.rs

```diff
@@ -95,6 +97,20 @@ impl<T> Event<T> {
             EventData::Added(_) => None,
         }
     }
+
+    /// Maps the inner data of an event into some other data.
+    pub fn map<U, F>(self, f: F) -> Event<U>
```

Comment:
Where is this method used?

Reply:
Its a utility method for `ethcontract` crate consumers. It's being used in my event example I am working on and I imagine it would be useful so I included it in the crate.